src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter dateFormat = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final int DATE_ARRAY_INDEX = 0;
+    private static final int NAME_ARRAY_INDEX = 1;
+    private static final int WORK_HOURS_ARRAY_INDEX = 2;
+    private static final int SALARY_PER_HOUR_ARRAY_INDEX = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder report = new StringBuilder();
+        try {
+            LocalDate fromDate = LocalDate.parse(dateFrom, dateFormat);
+            LocalDate toDate = LocalDate.parse(dateTo, dateFormat);
+
+            report
+                    .append("Report for period ")
+                    .append(dateFrom).append(" - ")
+                    .append(dateTo)
+                    .append(System.lineSeparator());
```

Comment:
not fixed prev comment: 

![image](https://github.com/mate-academy/jv-salary-info/assets/109764850/173ee39d-566e-4159-a70b-74c5ad804dfb)

```suggestion
                    .append("Report for period ")
                    .append(dateFrom)
                    .append(" - ")
                    .append(dateTo)
                    .append(System.lineSeparator());
```